### PR TITLE
Travis to GA

### PR DESCRIPTION
**Pull request by prisae**

### CODE REVIEW

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+      - name: Test with pytest
+        shell: bash -l {0}
+        run: |
+          python setup.py install
+          cp ./tests/matplotlibrc ./.
+          pytest --cov=empymod --flake8 ${{ matrix.case.test }}
+
+      - name: Coveralls
+        # [pin v20201129]
+        uses: AndreMiras/coveralls-python-action@f5fd5c309b39d01599fb92c72d4f7409ea78aec9
```
Comment (AndreMiras) [inline .github/workflows/pytest.yml:131]: Why not using the tag directly?
```yaml
        uses: AndreMiras/coveralls-python-action@v20201129
```

Comment (prisae) [reply .github/workflows/pytest.yml:131]: For security reasons. If your GitHub is compromised, someone could delete your release `v20201129`, create another release with the tag `v20201129`, and put malicious code in it. At least that is how I understood this:

https://github.com/pypa/gh-action-pypi-publish/issues/27

I got a lot of my Action-yaml-ideas from https://github.com/fatiando/boule/blob/master/.github/workflows/pypi.yml - so @leouieda looked probably deeper into this issue and will correct me ;-)

Comment (prisae) [reply .github/workflows/pytest.yml:131]: I guess the important / more extensive link is in the comment of the GitHub issue I posted: https://julienrenaux.fr/2019/12/20/github-actions-security-risk/

Comment (leouieda) [reply .github/workflows/pytest.yml:131]: Yeah, I’m a bit paranoid 🙂 I don’t much like the idea of giving a token to any action author out there. Not that I think you’re malicious @andremiras. But there are a lot of actions and I find it worrisome that GitHub has no control over this. It’s inevitable that someone will make an action at some point that steals tokens.

Comment (AndreMiras) [reply .github/workflows/pytest.yml:131]: I understand your concern, isn't computer security all about being paranoid?
It's also about finding the right balance between security and productivity while keeping in mind the risk (impact/likelihood).
I for now prefer to keep it simple and readable with the tag, but I'm happy this potential flaw got brought to my attention :smiley:

Comment (leouieda) [reply .github/workflows/pytest.yml:131]: You're absolutely right about needing a balance. For me, I've decided to use the commit hash whenever giving a token to an Action. I'm also not too worried about the official GitHub actions.

**Pull request merged**